lib/iris/fileformats/pp_save_rules.py

```diff
@@ -845,7 +845,7 @@ def _vertical_rules(cube, pp):
 
 def _all_other_rules(cube, pp):
     """
-    Rules for setting the horizontal grid and pole location of the PP field.
+    Rules for setting the field code and ensemble member number of the PP field.
```

Comment:
Given the wide scope of the name `_all_other_rules`, I think we should be careful with statements that sound categorical. Would be better to have something like `Fields currently managed by these rules: <list>`.